tests/Makefile.am

```diff
@@ -94,18 +92,21 @@ YAMI_VPP_CFLAGS = \
 	$(NULL)
 
 yamidecode_LDADD    = $(YAMI_VPP_LIBS)
-yamidecode_LDFLAGS  = $(YAMI_VPP_LDFLAGS)
+yamidecode_CPPFLAGS = $(YAMI_COMMON_CFLAGS)
+yamidecode_LDFLAGS  = $(YAMI_VPP_LDFLAGS) $(AM_LDFLAGS)
 yamidecode_SOURCES  = decode.cpp decodehelp.cpp $(DECODE_INPUT_SOURCES) decodeoutput.cpp vppinputoutput.cpp vppinputdecode.cpp vppoutputencode.cpp encodeinput.cpp encodeInputCamera.cpp encodeInputDecoder.cpp vppinputdecodecapi.cpp md5.c
 if ENABLE_EGL
 yamidecode_SOURCES += ../egl/egl_util.c ./egl/gles2_help.c
 endif
 
 yamiencode_LDADD    = $(YAMI_ENCODE_LIBS)
-yamiencode_LDFLAGS  = $(YAMI_ENCODE_LDFLAGS)
+yamiencode_CPPFLAGS = $(YAMI_COMMON_CFLAGS)
+yamiencode_LDFLAGS  = $(YAMI_ENCODE_LDFLAGS) $(AM_LDFLAGS)
```

Comment:
`$(YAMI_ENCODE_LDFLAGS)` doesn't appear to be defined anywhere.  Thus, this override can be removed.